client/my-sites/plan-features/actions.jsx

```diff
@@ -69,7 +70,14 @@ const PlanFeaturesActions = ( {
 		if ( isLandingPage ) {
 			buttonText = translate( 'Select', { context: 'button' } );
 		}
-		if ( isInSignup ) {
+		if ( isLaunchPage ) {
+			if ( freePlan ) {
```

Comment:
Would it still work if we were to combine the conditions into one statement?

```
if ( isLaunchPage && freePlan ) {
```

Reply:
I've fixed this in [fa5d7ad](https://github.com/Automattic/wp-calypso/pull/30242/commits/fa5d7ad3e87b2d3da2cff9d839e1193845dcb35c) which adds the else part to the logic( button text as suggested by Dan [in this comment](https://github.com/Automattic/wp-calypso/pull/30242#issuecomment-458991265)).

([5a4966e](https://github.com/Automattic/wp-calypso/pull/30242/commits/5a4966e655218512c62b2e0fa69870e72897899e) fixes the typo in the above commit)